internal/provisioner/kops_utils.go

```diff
@@ -113,14 +113,28 @@ func getLoadBalancerEndpoint(ctx context.Context, namespace string, logger log.F
 }
 
 // installHelmChart is used to install Helm charts.
-func installHelmChart(chart helmDeployment, configPath string) error {
+func installHelmChart(chart helmDeployment, configPath string, logger log.FieldLogger) error {
 	if chart.setArgument != "" {
-		err := exec.Command("helm", "install", "--kubeconfig", configPath, "--set", chart.setArgument, "-f", chart.valuesPath, chart.chartName, "--namespace", chart.namespace, "--name", chart.chartDeploymentName).Run()
+		cmd := exec.Command("helm", "--debug", "install", "--kubeconfig", configPath, "--set", chart.setArgument, "-f", chart.valuesPath, chart.chartName, "--namespace", chart.namespace, "--name", chart.chartDeploymentName)
+
+		logger.WithFields(log.Fields{
+			"cmd":  cmd.Path,
+			"args": cmd.Args,
+		}).Info("Invoking command")
+
+		err := cmd.Run()
 		if err != nil {
 			return err
 		}
 	} else {
-		err := exec.Command("helm", "install", "--kubeconfig", configPath, "-f", chart.valuesPath, chart.chartName, "--namespace", chart.namespace, "--name", chart.chartDeploymentName).Run()
+		cmd := exec.Command("helm", "--debug", "install", "--kubeconfig", configPath, "-f", chart.valuesPath, chart.chartName, "--namespace", chart.namespace, "--name", chart.chartDeploymentName)
+
+		logger.WithFields(log.Fields{
```

Comment:
Instead of defining and executing the `cmd`s within the if loop, we could run the command outside of the loop to streamline the code a bit.

Reply:
Yeah, I see what you're saying. I've cleaned this up a bit.